nucleus/src/main/java/za/co/absa/subatomic/adapter/team/rest/TeamController.java

```diff
@@ -70,8 +70,9 @@ public TeamController(TeamService teamService, TeamPersistenceHandler teamPersis
         return ResponseEntity.created(location).build();
     }
 
-    @PutMapping("/{id}")
+    @PutMapping("/{id}/members/{memberId}")
     ResponseEntity<TeamResource> update(@PathVariable String id,
+                                        @PathVariable String memberId,
```

Comment:
Dont add this path variable. The member id stuff is already included in the TeamResource.